packages/desktop-gui/src/lib/ipc-bus.js

```diff
@@ -62,6 +62,18 @@ const ipcBus = (...args) => {
   // get the last argument
   const lastArg = args.pop()
 
+  // remove nodes that cannot be cloned
+  // https://github.com/cypress-io/cypress/issues/6750
+  args = args.map((arg) => {
```

Comment:
is there a test for this?

Reply:
no, i can add tests, just wasn't sure where to add them... probably just need to add tests for the `ipc-bus` file

Reply:
it would be nice to send an object from the browser with functions and elements and ping it back from the server - and the receiver would check if the returned object has invalid fields stripped

Reply:
added ipc_bus specs: https://github.com/cypress-io/cypress/pull/6753/files#diff-0bf397162937f0ddea8c95e4dd3e3033